controllers/propagator/placement_change_reconclier.go

```diff
@@ -0,0 +1,190 @@
+package propagator
+
+import (
+	"context"
+	"sync"
+
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	clusterv1beta1 "open-cluster-management.io/api/cluster/v1beta1"
+	appsv1 "open-cluster-management.io/multicloud-operators-subscription/pkg/apis/apps/placementrule/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	policiesv1 "open-cluster-management.io/governance-policy-propagator/api/v1"
+	"open-cluster-management.io/governance-policy-propagator/controllers/common"
+)
+
+//+kubebuilder:rbac:groups=policy.open-cluster-management.io,resources=placementbindings,verbs=get;list;watch;create;update;patch;delete
```

Comment:
I think you'll need to add placement decision here

Reply:
I debated not splitting the kubebuilder markers at all - these all get considered together when making the ClusterRole, so it doesn't really make a difference where they are. Or maybe I should split it more, so that the propagator has its required permissions separately listed?

If I split [this marker](https://github.com/open-cluster-management-io/governance-policy-propagator/blob/4178f22748c05eafb42a4b2ed2fe72391a319d70/controllers/propagator/policy_controller.go#L38C1-L39C1) so that the placement-change-reconciler gets the `placement` and `placementdecision` parts, then the generated ClusterRole will have separate entries for those and `managedclusters`. I was trying to skip the work of updating the role in charts and things.